src/main/java/de/tum/in/www1/artemis/service/metis/ConversationMessagingService.java

```diff
@@ -236,24 +236,24 @@ private static Set<User> mapToUsers(Set<ConversationNotificationRecipientSummary
      * Filters the given list of recipients for users that should receive a notification about a new message.
      * <p>
      * In all cases, the author will be filtered out.
-     * If the conversation is not an announcement channel, the method filters out participants, that have hidden the conversation.
+     * If the conversation is not an announcement channel, the method filters out participants, that have muted or hidden the conversation.
      * If the conversation is not visible to students, the method also filters out students from the provided list of recipients.
      *
-     * @param author              the author of the message
-     * @param conversation        the conversation the new message has been written in
-     * @param webSocketRecipients the list of users that should be filtered
-     * @param mentionedUsers      users mentioned within the message
+     * @param author                 the author of the message
+     * @param conversation           the conversation the new message has been written in
+     * @param notificationRecipients the list of users that should be filtered
+     * @param mentionedUsers         users mentioned within the message
      * @return filtered list of users that are supposed to receive a notification
      */
-    private Set<User> filterNotificationRecipients(User author, Conversation conversation, Set<ConversationNotificationRecipientSummary> webSocketRecipients,
+    private Set<User> filterNotificationRecipients(User author, Conversation conversation, Set<ConversationNotificationRecipientSummary> notificationRecipients,
             Set<User> mentionedUsers) {
         // Initialize filter with check for author
         Predicate<ConversationNotificationRecipientSummary> filter = recipientSummary -> !Objects.equals(recipientSummary.userId(), author.getId());
 
         if (conversation instanceof Channel channel) {
-            // If a channel is not an announcement channel, filter out users, that hid the conversation
+            // If a channel is not an announcement channel, filter out users, that muted or hid the conversation
             if (!channel.getIsAnnouncementChannel()) {
-                filter = filter.and(summary -> !summary.isConversationHidden() || mentionedUsers
+                filter = filter.and(summary -> ((!summary.isConversationMuted() && !summary.isConversationHidden())) || mentionedUsers
```

Comment:
This would be better as a property on the summer: summary.shouldUserBeNotified()

Reply:
Thanks, will do